src/components/Organisms/AssetMeta/AssetMeta.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { Card } from '@/components/Atoms/Card'
+import { Icon } from '@/components/Atoms/Icon'
+import { Tag } from '@/components/Atoms/Tag'
+import { FlexBox, FlexItem } from '@/components/Helper/FlexBox'
+import { FeatureList, FeatureListType } from '@/components/Molecules/FeatureList'
+import {
+  SvgBookmark,
+  SvgBookmarkOutlined,
+  SvgCheckmarkCircleOutlined,
+  SvgSaveToPlaylist,
+  SvgShare,
+} from '@/svgs'
+import { CompetencyText, CompleteButton, IconsWrapper, LaunchButton } from './AssetMeta.styled'
+
+export interface AssetMetaProps {
+  isShareInteraction: boolean
+  isBookmarkInteraction: boolean
+  isCreatePlaylistInteraction: boolean
+  buttonTextLanch: string
+  buttonTextComplete: string
+  list: FeatureListType
+  competencyCopy: string
+  tags: { label: string; href: string }[]
+}
+
+export const AssetMeta: React.FC<AssetMetaProps> = ({
+  isShareInteraction,
+  isBookmarkInteraction,
+  isCreatePlaylistInteraction,
+  buttonTextLanch,
+  buttonTextComplete,
+  list,
+  competencyCopy,
+  tags,
+}): JSX.Element => {
+  return (
+    <FlexBox flexDirection="column" alignItems="center">
+      <IconsWrapper>
+        {isBookmarkInteraction && (
+          <FlexItem padding>
+            <Icon>
+              <SvgBookmarkOutlined />
+            </Icon>
+          </FlexItem>
+        )}
+        {isShareInteraction && (
+          <FlexItem padding>
+            <Icon>
+              <SvgShare />
+            </Icon>
+          </FlexItem>
+        )}
+        {isCreatePlaylistInteraction && (
+          <FlexItem padding>
+            <Icon>
+              <SvgSaveToPlaylist />
+            </Icon>
+          </FlexItem>
+        )}
+      </IconsWrapper>
+      <Card
+        elevation={1}
+        shape="rounded-small"
+        padding={{
+          bottom: 'lg',
+          left: 'lg',
+          right: 'lg',
+          top: 'lg',
+        }}>
+        <LaunchButton curved fullWidth>
+          <Icon width={25} height={25}>
+            <SvgBookmark />
+          </Icon>
+          <FlexItem>{buttonTextLanch}</FlexItem>
+        </LaunchButton>
+        <CompleteButton curved actionType="outlined" fullWidth>
+          <Icon width={25} height={25}>
+            <SvgCheckmarkCircleOutlined />
+          </Icon>
+          {buttonTextComplete}
+        </CompleteButton>
+        <FeatureList list={list} />
+        <CompetencyText color="grey4">{competencyCopy}</CompetencyText>
+        <FlexBox flexWrap="wrap">
+          {tags.map((tag, index: number) => (
+            <FlexItem key={index}>
+              <Tag
+                curved
+                color="primary"
+                marginBottom
+                marginRight
+                text={tag.label}
```

Comment:
Would be good to keep the types synced, as TagProps is expecting the tag to have a text or href
```suggestion
                text={tag.text}
```